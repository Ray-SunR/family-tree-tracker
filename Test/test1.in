! A
+ A B
+ A C
+ A D


+ A E
+ A F
+ A G
+ A H
+ A I
+ A J
? A
? B
? C
? D
p
