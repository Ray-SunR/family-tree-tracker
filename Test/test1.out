A
A,B
A,C
A,D
A is parent of B,C,D,E,F,G,H,I,J
